src/django/api/migrations/0216_backfill_moderation_event_os_id_snapshot.py

```
@@ -0,0 +1,33 @@
+    ModerationEvent.objects.filter(
+        status='APPROVED',
+        os_id_snapshot='',
+        os_id__isnull=False,
+    ).update(os_id_snapshot=F('os_id'))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0213_add_os_id_snapshot_to_moderation_event'),
+    ]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)